.vscode/tasks.json

```diff
@@ -14,12 +14,12 @@
     {
       "label": "run-api-tests-and-coverage",
       "type": "shell",
-      "command": "${command:python.interpreterPath} -m pytest -v --cov-report xml:cov.xml --cov . ./api/tests/"
+      "command": "${command:python.interpreterPath} -m pytest -v --cov-report html --cov . ./api/tests/"
```

Comment:
Do we have a better line coverage tool with this html format? The cov.xml is being read by the koverage tool (and coverage gutters) to show us line coverage within vscode.

Reply:
Oh, I had no clue how to view the XML coverage reports. Just tried those extensions and those are a much better way to do it, I'll revert that change.